signs-work-order-map/types/map.ts

```diff
@@ -0,0 +1,107 @@
+export interface LatLon {
+  latitude: number | undefined;
+  longitude: number | undefined;
+}
+
+export interface MapProps {
+  location: {
+    longitude: number | undefined;
+    latitude: number | undefined;
+  };
+  signs: Sign[];
+}
+
+export interface Sign {
+  id: string;
+  lng: number;
+  lat: number;
+  spatialId: number;
+  workOrderId: string;
+  isLocationDetailPage: boolean;
+}
+
+export interface KnackRecord {
+  /** spatial id */
+  field_3297: number;
+  /** spatial id raw format */
+  field_3297_raw: number;
+  /** address in string form */
+  field_3300: string;
+  /** address raw format  */
+  field_3300_raw: {
+    city: string;
+    country: string;
+    full: string;
+    latitude: number;
+    longitude: number;
+    state: string;
+    street: string;
+    street2: string | null;
+    zip: string;
+  };
+  /** created date */
+  field_3302: string;
+  /** created date raw format */
+  field_3302_raw: {
+    am_pm: "AM" | "PM";
+    date: string;
+    date_formatted: string;
+    hours: string;
+    iso_timestamp: string;
+    minutes: string;
+    proper_iso_timestamp: string;
+    proper_unix_timestamp: number;
+    time: number;
+    timestamp: string;
+    unix_timestamp: number;
+  };
+  /** photo */
+  field_3378: string;
+  /** photo raw format */
+  field_3378_raw: string;
+  /** count of assets  */
+  field_3425: number;
+  /** count of assets raw format*/
+  field_3425_raw: number;
+  /** knack record id */
+  id: string;
+}
```

Comment:
Are we sure that all of these fields will all be defined? To err on the side of caution you could make them all optional (`?`)